service/core.go

```diff
@@ -64,19 +66,20 @@ type SetContainerer interface {
 }
 
 type serviceCore struct {
-	standardConfig serviceConfig
-	roles          []string
-	state          State
-	configProvider config.Provider
-	scopeMux       sync.Mutex
-	scope          tally.RootScope
-	observer       Observer
-	resources      map[string]interface{}
-	logConfig      ulog.Configuration
-	log            ulog.Log
-	tracerConfig   jaegerconfig.Configuration
-	tracer         opentracing.Tracer
-	tracerCloser   io.Closer
+	standardConfig  serviceConfig
+	roles           []string
+	state           State
+	configProvider  config.Provider
+	scopeMux        sync.Mutex
+	scope           tally.RootScope
+	runtimeReporter *metrics.RuntimeReporter
+	observer        Observer
+	resources       map[string]interface{}
+	logConfig       ulog.Configuration
+	log             ulog.Log
+	tracerConfig    jaegerconfig.Configuration
+	tracer          opentracing.Tracer
+	tracerCloser    io.Closer
```

Comment:
if we're going to be messing up history here can we alphabetize these? this is getting pretty big and hard to read